e2e/scenario/base_vhd_catalog.json

```diff
@@ -1,41 +1,48 @@
 {
     "ubuntu1804": {
         "gen2containerd": {
-            "artifactName": "1804-gen2-containerd",
-            "resourceId": "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/1804Gen2/versions/1.1716223055.6413"
+            "versionTagName": "branch",
+            "versionTagValue": "r2k1/e2e-update",
```

Comment:
alternatively, I can make a single variable for all images.

Set it default to master, with an option to override though ENV variable.
Like `TAG=buildId:12346`. (And with optional fallback to master if the version doesn't exist)